rfcs/convergence/first-rule-of-aria.md

```diff
@@ -0,0 +1,204 @@
+
+## Problem statement
+
+The first rule of accessibility is that it is impossible to predict how a user will attempt to interact with your interface. We test some basics: screen readers, static zoom, high contrast mode, and keyboard access, but this utterly fails to capture the wide variety of tools and access methods used in the real world. From extreme red light filters to custom switch setups to a browser's reader mode to eye gaze and voice control, what access looks like in practice is often messy and unpredictable. Disabilities often overlap, and assistive tech is frequently used in ways we do not test for.
+
+As a very limited example, we currently test a few common screen readers under the assumption that the people using them are blind, on a modern device, and reasonably proficient with the software. Here are some ways people use screen readers that do not fit our current test criteria:
+
+- As a supplement to low vision, used primarily to read text, while sight + mixed mouse & keyboard input is still used to understand and navigate the UI
+- To assist in reading and navigating by someone who has perfect vision, but has migraines that are exacerbated by concentrating on screens
+- To read text by someone with perfect vision, as an aid for a cognitive disability like dyslexia
+- With a braille display, by someone who is deafblind
+- With a screen magnifier, e.g. ZoomText Fusion
+
+None of these use cases are particularly unusual, and there are certainly many more uses that I'm not personally aware of. And there is even more diversity found in assistive tech beyond screen readers. On the more well-known end, we have tools like voice control software, zoom/text enlarging, Windows High Contrast Mode and other color-altering software, and switch devices. But then we also have tools like Edge's Immersive Reader, browser extensions to pause auto-playing gifs and videos, eye control, and even adblockers are a form of assistive tech to the right person. Each of these can be just as vital for accessing the web as screen readers are.
+
+All this diversity of interaction introduces the problem of how to support it, particularly since it's impossible to test everything. As a general rule, the best way to create robustly accessible UI is to rely on features and interaction built into the web platform. This essentially means using semantic HTML whenever possible, and relying on browsers to translate that UI into a usable experience across platforms and tech. As soon as we create a custom interactive control with ARIA instead of HTML, we take on full responsibility for ensuring that interaction works across all access tools. That task is often difficult, and sometimes impossible.
+
+Some examples of where `<div>`s + ARIA fall down vs. semantic HTML include:
+
+- Some of the most popular voice control software has very limited support for ARIA
+- Windows High Contrast Mode only assigns color based on HTML semantics, not ARIA
```

Comment:
High contrast should be primarily responsibility of the theme, even including system colors (https://www.w3.org/TR/css-color-4/#css-system-colors)

Reply:
If we use semantic elements, we often don't need to have any custom styles for WHCM, which also means we don't need to worry about support for the `-ms` media query vs. `forced-colors`, or updates in the color keywords :)